### PR TITLE
Load in second floor rooms for public Gahreesens

### PR DESCRIPTION
**Pull request by EhrenCG**
A visual fix to hood Gahreesens, always load second floor (not including gear room).  It always bothered me how you could clearly see the second floor is missing by looking up into the window from the link-in-point area.

### CODE REVIEW

**Comment by Deledrius** [comment]
Good catch.  I'll try this out as soon as I can.  
BTW, these commits should be squashed into a single commit.  If you need assistance with that, feel free to ask. :smile:

File: Python/grsnPageMaster.py [hunk 1]
```diff
@@ -88,10 +88,12 @@ def __init__(self):
 
         # for public version, add any pages specific only for that
         if IsPublic:
-            pages += ["WellWindowFake"]        
+            #this page is blank, so always load the second floors page instead
+            #pages += ["WellWindowFake"]
```
Comment (Hoikas) [inline Python/grsnPageMaster.py]: This is version control--no need to comment it out. Just nuke it.

**Comment by EhrenCG** [comment]
Yeah, I guess I do need help, the format and terms here are a bit more complicated than I expected, so I don't know how to squash commits together.  Now I made a change to one, but it seems to count as yet another new commit, not sure if that means I did something wrong again.

**Comment by Deledrius** [comment]
Not wrong, just a bit more cluttered than we prefer for history we pull in.  Are you editing the files directly in your browser on GitHub?

**Comment by EhrenCG** [comment]
Yes, I directly used a browser.

**Comment by Deledrius** [comment]
Alright, don't worry about squashing them then.  I don't believe there's a way for you to clean them up through the web interface.  I can do it if this is accepted.  It looks fine to me, and I'll be testing it shortly.

**Comment by Deledrius** [comment]
Works for me, and seems like an adequate fix.  Loading those extra rooms shouldn't slow down people too much.  Long-term, I'd prefer to build a replacement for the missing `WellWindowFake` page, but your fix allows us to have a better visual experience immediately.  :+1:

**Comment by Deledrius** [comment]
Closed by 75fbaea051b4ae2352df6cc9cd25b7b4fbe20a24

Thanks, @EhrenCG!

**Pull request closed without merging**